README.md

```diff
@@ -117,3 +119,7 @@ clean | delete everything listed in .gitignore
     make source
     make -j metacheck
     make -j html htmlhelp chm
+
+## License
+
+Source code in this repository is distributed under MIT license. See [`LICENSE`](LICENSE) for more information. Manual content follows [Adventure Game Studio's license](https://github.com/ericoporto/ags/blob/ags3--sdl2--emscripten/License.txt).
```

Comment:
Could you adjust this link to point to the adventuregamestudio organisation?

I've looked for a generic method to link to the current licence for a repository instead of using a fixed path, I couldn't find one, so presumably the link has to be https://github.com/adventuregamestudio/ags/blob/master/License.txt

Reply:
Yes, I am sorry, I should adjust that!